src/database/DataTypes/MasterList.js

```diff
@@ -72,6 +72,14 @@ export class MasterList extends Realm.Object {
     this.addIndicator(programIndicator);
   }
 
+  /**
+   * Get all indicators currently active on this master list.
+   * @returns {Array.<ProgramIndicator>}
+   */
+  getActiveIndicators() {
```

Comment:
why is this not a `getter`?

Reply:
ef0a03bb4f1e9a9c5aff1de9b7688d6d00fb6cac.

![please](https://user-images.githubusercontent.com/43223637/72690407-0772b780-3b81-11ea-9c4a-4b0388a71749.gif)